src/utils.rs

```diff
@@ -512,7 +526,7 @@ pub fn format_url(url: &str) -> String {
 
 			match domain {
 				"v.redd.it" => chain!(
-					capture(r"https://v\.redd\.it/(.*)/DASH_([0-9]{2,4}(\.mp4|$))", "/vid/", 2),
+					capture(r"https://v\.redd\.it/(.*)/DASH_([0-9]{2,4}(\.mp4|$|\?source=fallback))", "/vid/", 2),
```

Comment:
Does this `source=fallback` query parameter change the video?

Reply:
What do you mean by change? I think it is the same as not having the parameter, but the reddit JSON that I have been using gave me this parameter without giving me a `.mp4` extension, which initially made `format_url` return an empty string (which lead me to write the tests for it, and also pull the reddit JSON for testing)

Reply:
That makes sense, thanks for catching that!